src/context/Router.tsx

```diff
@@ -1,8 +1,386 @@
+import { WorkflowContextProvider } from 'context/Workflow';
+import { OAuthPopup } from 'hooks/forks/react-use-oauth2/components';
+import useBrowserTitle from 'hooks/useBrowserTitle';
+import Admin from 'pages/Admin';
+import Auth from 'pages/Auth';
+import Captures from 'pages/Captures';
+import Collections from 'pages/Collections';
+import TestJsonForms from 'pages/dev/TestJsonForms';
+import PageNotFound from 'pages/error/PageNotFound';
+import Home from 'pages/Home';
+import Login from 'pages/Login';
+import Materializations from 'pages/Materializations';
+import {
+    createBrowserRouter,
+    createRoutesFromElements,
+    Route,
+    RouterProvider,
+} from 'react-router-dom';
+import { isProduction } from 'utils/env-utils';
+import { RequireAuth } from './Authenticated';
 
-const AppRouter = ({ children }: BaseComponentProps) => {
-    return <BrowserRouter>{children}</BrowserRouter>;
+const router = createBrowserRouter(
+    createRoutesFromElements(
+        <Route>
+            <Route
+                path={unauthenticatedRoutes.path}
+                element={
+                    <RequireAuth firstLoad>
+                        <Login />
+                    </RequireAuth>
+                }
+            />
+
+            <Route
+                path={unauthenticatedRoutes.login.path}
+                element={
+                    <RequireAuth firstLoad>
+                        <Login />
+                    </RequireAuth>
+                }
+            />
+
+            <Route path={unauthenticatedRoutes.auth.path} element={<Auth />} />
+            <Route
+                path={unauthenticatedRoutes.magicLink.path}
+                element={<Auth />}
+            />
+            <Route
+                path={unauthenticatedRoutes.register.path}
+                element={<Login showRegistration />}
+            />
+
+            {/* This is not in the route below so that it does not include the applayout*/}
+            <Route
+                path={authenticatedRoutes.oauth.path}
+                element={
+                    <AuthenticatedOnlyContext>
+                        <OAuthPopup />
+                    </AuthenticatedOnlyContext>
+                }
+            />
+
+            <Route
+                path={authenticatedRoutes.path}
+                element={
+                    <AuthenticatedOnlyContext>
+                        <AppLayout />
+                    </AuthenticatedOnlyContext>
+                }
+            >
+                <Route>
+                    <Route
+                        path={authenticatedRoutes.home.path}
+                        element={<Home />}
+                    />
+
+                    <Route
+                        path={authenticatedRoutes.home.path}
+                        element={<Home />}
+                    />
+
+                    <Route path={authenticatedRoutes.collections.path}>
+                        <Route
+                            path=""
```

Comment:
These now need to be blank. However, using a path param gets confusing so just hardcoding. Not a big deal as we'll be moving to the JSON routes approach soon.